### ISSUE TITLE
[BUG] End Troll Boxes losing Items

### ISSUE THREAD

**Issue by Xaikii** [andrew0030/Pandoras-Creatures#28]
**Describe the bug**
End Troll Boxes will lose some of the items if opened in hand.

**To Reproduce**
1. Get a Troll Box and place it
2. Fill 4 or more rows with items
3. Break it
4. Open it in hand
5. Place it again

**Expected behavior**
The items that couldn't be shown while opened in hand still being in the Box.

**Screenshots**
N/A

**Debug crash log**
N/A

**Versions**
Pandora's Creatures: 2.0.1
Forge: 35.1.7

**Additional context**
The items only disappear if the Box had been opened in hand.
Breaking and replacing it wont make the items disappear, but opening it will.

**Comment by andrew0030** [comment]
Opened in hand?

**Comment by Xaikii** [comment]
Is that not supposed to be a feature?
If so another mod seems to interfere here.

**Comment by andrew0030** [comment]
No they are like shulker boxes, you open them by placing them

**Comment by Xaikii** [comment]
Ok, then it seems like this is an issue of cross-mod interaction going wrong, huh?
https://www.curseforge.com/minecraft/mc-mods/advanced-shulkerboxes

**Comment by andrew0030** [comment]
Yeah from the looks of it that other mod is only intended to work with shulker boxes

**Comment by Xaikii** [comment]
Good to know, thanks for the help
